octopus_deploy/datadog_checks/octopus_deploy/check.py

```diff
@@ -82,18 +130,8 @@ def _get_new_tasks_for_project(self, project):
             queue_time = start_time_converted - queue_time_converted
             queue_time_seconds = queue_time.total_seconds()
 
-            if completed_time_converted > new_completed_time:
-                new_completed_time = completed_time_converted
-
             succeeded = int(state == DEPLOY_SUCCESS_STATE)
```

Comment:
I think there's a `FinishedSuccessfully` field in the response for this

Reply:
Updated!